core/src/test/java/org/frankframework/configuration/digester/ValidateAttributeRuleTest.java

```diff
@@ -497,6 +511,31 @@ public void setDeprecatedConfigWarningString(String str) {
 		public void setTestSuppressAttribute(String test) {
 			testString = test;
 		}
+
+		public enum QueryType {
+			OTHER,
+			/** Deprecated: Use OTHER instead */
+			@ConfigurationWarning("Use queryType 'OTHER' instead")
+			@Deprecated(since = "8.1") INSERT,
+		}
+	}
+
+	public static abstract class ExtendsClassWithEnum {
```

Comment:
Waarom bestaat deze ExtendsClassWithEnum?

Reply:
Ah, ik zie dat ik `QueryType` per ongeluk had laten staan in de class ClassWithEnum. Ik wilde die alleen zetten in de ExtendsClassWithEnum, om zo aan te tonen dat het met overerving ook gewoon werkt.